### PR TITLE
Add DeleteIngredientCommand 

### LINKED ISSUE #9: As a new user, I can delete the expired ingredient
so that I can remove the expired ingredient from the fridge

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteIngredientCommand.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.recipe.Ingredient;
+
+/**
+ * Deletes an ingredient identified using it's displayed index from the address book.
```
Comment (caitlinjee) [inline src/main/java/seedu/address/logic/commands/DeleteIngredientCommand.java]: I think it should be fridge!

Comment (oliviajohansen) [reply src/main/java/seedu/address/logic/commands/DeleteIngredientCommand.java]: ok! thanks

**Review by caitlinjee — APPROVED** [review]
lgtm

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Review by jiaqi20 — APPROVED** [review]
lgtm

**Review by daongochieu2810 — APPROVED** [review]

**Pull request merged**